schema/postgresql/v96/temporal/schema.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE namespaces(
   partition_id INTEGER NOT NULL,
   id BYTEA NOT NULL,
-  name VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
Do you plan to update for MySQL schema as well? I remember TEXT can not be part of index, or is that no longer the case?

Reply:
Yes, this is still in draft and I wanted to just keep a remote branch for tracking right now.

I am still digging into mysql, I will double check that constraint before submitting this as ready for review.

Reply:
Yeah, looks like in mysql text columns can be used but with a fixed length prefix. Which means that text is not really a good candidate for unique constraints for indexes.